nipype/interfaces/slicer/generate_classes.py

```diff
@@ -33,28 +33,28 @@ def force_to_valid_python_variable_name(old_name):
 
 def add_class_to_package(class_codes, class_names, module_name, package_dir):
     module_python_filename = os.path.join(package_dir, "%s.py" % module_name)
-    f_m = open(module_python_filename, "w")
-    f_i = open(os.path.join(package_dir, "__init__.py"), "a+")
-    f_m.write(
-        """# -*- coding: utf-8 -*-
+    with (
+        open(module_python_filename, "w") as f_m,
+        open(os.path.join(package_dir, "__init__.py"), "a+") as f_i,
+    ):
```

Comment:
There's nothing that requires these to be open simultaneously.

```suggestion
    with open(module_python_filename, "w") as f_m:
```